index.js

```diff
@@ -125,7 +123,15 @@ Download.prototype.run = function (cb) {
 		if (protocol) {
 			protocol = protocol.slice(0, -1);
 		}
-		var agent = caw(this.opts.proxy, {protocol: protocol});
+		var agent;
+		if(conf['strict-ssl'] === false){
+			agent = caw(objectAssign({
+				protocol: protocol,
+				rejectUnauthorized:false
```

Comment:
As my understanding `rejectUnauthorized` is not the one of `agent` properties. It should be directly passed to `https.request`.  https://nodejs.org/api/https.html#https_https_request_options_callback